basic/code1.s

```diff
@@ -200,7 +200,13 @@ chead	ldy #1
 	beq lnkrts
 	ldy #4
 czloop	iny
-	lda (index),y
+	bne :+		; Continue as normal as long as Y has not rolled over to 0
+	tya		; If Y has rolled over to 0
+	sta (index),y	; store 0's in the two first bytes of the txttab buffer
+	iny		; to show that there is no BASIC program
+	sta (index),y
+	bra lnkrts	; exit function (better with just an RTS?)
```

Comment:
I think this is exactly what we want. There's always the possibility that we restored garbage as one or more lines of an actual program.  (index) could be further along than at the beginning, but we still want to double-null terminate where we found the "endless" line.